governance/xc_admin/packages/xc_admin_common/src/contracts/Contract.ts

```diff
@@ -0,0 +1,139 @@
+import { ChainId, Instruction } from "@pythnetwork/xc-governance-sdk";
+import { ethers } from "ethers";
+
+export enum ContractType {
+  Oracle,
+  EvmPythUpgradable,
+  EvmWormholeReceiver,
+}
+
+/**
+ * A unique identifier for a blockchain. Note that we cannot use ChainId for this, as ChainId currently reuses
+ * some ids across mainnet / testnet chains (e.g., ethereum goerli has the same id as ethereum mainnet).
+ */
+export type NetworkId = string;
```

Comment:
Cartesian product of wormhole network + chainid?

Reply:
we could use that tuple, but it's going to get confusing, especially as we start rolling out stable/edge configurations to each network. In that case ethereum goerli stable will be `[ethereum_goerli, mainnet]` and ethereum goerli edge will be `[ethereum, testnet]`. I think it's better to make a clean break here and define the right concept.